meetings/2018-01-10.md

```diff
@@ -0,0 +1,117 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2018-01-10
+
+## Links
+* **Recording**: https://www.youtube.com/watch?v=StjmGRclNi0
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/465
+* **Minutes Google Doc**: https://docs.google.com/document/d/1ShVcDk_-BzwIXOhoV0iECpjJZ53u6-GJrcEc7K1YMqI
+
+## Present
+* Anna Henningsen @addaleax (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Franziska Hinkelmann @fhinkel (TSC)
+* Colin Ihrig @cjihrig (TSC)
+* Evan Lucas @evanlucas (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* Gibson Fahnestock @gibfahn (observer)
+* James M Snell @jasnell (TSC)
+* Joyee Cheung @joyeecheung (TSC)
+* Matteo Collina @mcollina (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Ali Ijaz Sheikh @ofrobots (TSC)
+* Michaël Zasso @targos (TSC)
+* Sakthipriyan Vairamani @thefourtheye (TSC)
+* Rich Trott @Trott (TSC)
+
+## Agenda
+### Announcements
+* 8.9.4 out
+* Planning for 10.X is out. Try to plan major changes before end of Feb as we'll cut
+  10.x branch then.
+* Release group is discussing creating a more concrete schedule for releases  
+
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/TSC
+* Mentorship initiative ramping up. [#443](https://github.com/nodejs/TSC/issues/443)
+  * Just wanted to make sure everybody is aware
+* Enterprise Advisory Group [#431](https://github.com/nodejs/TSC/issues/431)
+  * James: Heads up at this point. Goal is to look at things happening core and provide
+    feedback. TSC members to be invited as observers as opposed to members.
+  * James: Goal is to bring in companies who have made significant contributions to Node.js
+    and get them a channel for giving their input.
+  * Franziska: who is responsible for receiving and acting on the feedback.
+  * James: will attend as observer and will provide feedback, other TSC members are 
+    free to participate as observer as well.
+ * Franziska: should we clarify in the charter that there is no commitment on the TSC's part.
```

Comment:
The indentation looks off here, this (and subsequent) bullet(s) are still part of the Enterprise Advisory Group discussion.

Reply:
Pushed commit to fix.